pkg/odo/cli/env/view.go

```diff
@@ -0,0 +1,107 @@
+package env
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"text/tabwriter"
+
+	"github.com/openshift/odo/pkg/envinfo"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubectl/pkg/util/templates"
+)
+
+const viewCommandName = "view"
+
+var (
+	viewLongDesc = ktemplates.LongDesc(`
+	View current values in odo environment file
+	`)
+
+	viewExample = ktemplates.Examples(`
+	# For viewing the current environment configuration settings
+	%[1]s
+	`)
+)
+
+// ViewOptions encapsulates the options for the command
+type ViewOptions struct {
+	context string
+	cfg     *envinfo.EnvSpecificInfo
+}
+
+// NewViewOptions creates a new ViewOptions instance
+func NewViewOptions() *ViewOptions {
+	return &ViewOptions{}
+}
+
+// Complete completes ViewOptions after they've been created
+func (o *ViewOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	o.cfg, err = envinfo.NewEnvSpecificInfo(o.context)
+	if err != nil {
+		return errors.Wrap(err, "failed to load environment file")
+	}
+
+	return nil
+}
+
+// Validate validates the ViewOptions based on completed values
+func (o *ViewOptions) Validate() (err error) {
+	if !o.cfg.EnvInfoFileExists() {
+		return errors.Errorf("the context directory doesn't contain a component, please refer `odo create --help` to create a component")
```

Comment:
"Please refer `odo create --help` on how to create a component"

Reply:
I can update the message in my next commit